src/Ledger/PParams.lagda

```diff
@@ -385,13 +388,14 @@ to the general purpose that each parameter serves.
 \end{itemize}
 
 The first four groups have the property that every protocol parameter
-is associated to precisely one of these groups. The \SecurityGroup is
-special: a protocol parameter may or may not be in the
-\SecurityGroup. So, each protocol parameter belongs to at least one
-and at most two groups. Note that in \cite{cip1694} there is no
-\SecurityGroup, but there is the concept of security-relevant protocol
-parameters. The difference between these notions is only social, so we
-implement security-relevant protocol parameters as a group.
+is associated to precisely one of these groups.  The \SecurityGroup is
+special: a protocol parameter may or may not be in the \SecurityGroup.
+So, each protocol parameter belongs to at least one and at most two groups.
+Note that in \href{https://cips.cardano.org/cip/CIP-1694}{CIP-1694}
+there is no \SecurityGroup, but there is the concept of security-relevant
+protocol parameters~\cite{cip1694}.  The difference between these notions
+is only social, so we implement security-relevant protocol parameters as
+a group.
```

Comment:
Is strange to use both \href to refer to a CIP and a citation. We should unify this.

Reply:
I don't think it's strange.  I think it's nicer to have occurrences of "CIP-1694" link directly to the CIP, but also include the more conventional reference.  Sometimes you want to navigate directly to the CIP and sometimes you just want to look the bib metadata.  These serve two different purposes.

But I do think it's a good idea to have a latex macro named `\cip1694` to make it easier to be consistent whenever we refer to \cip1694.

Reply:
Since TeX doesn't allow numbers in macro names, I've used the "leet speak" equivalents, so, for example, \citeIGgA in a tex or agda source file will be typeset as CIP-1694 with a hyperlink to the CIP.

I'm not particularly fond of leet speak, so if you have a better suggestion for naming the macros, please propose it.

Reply:
I'd suggest in any case to use a command that gets as parameter the CIP not a command per CIP. For example,
```latex
\ExplSyntaxOn
\NewDocumentCommand\hrefCIP{m}{%
  \clist_if_in:nnTF {0069,1694} {#1}
  {%
    \href{https://github.com/cardano-foundation/CIPs/tree/master/CIP-#1}{CIP-#1}%
  }{%
    \PackageError{agda-latex-macros}{CIP-#1~undefined.}{}
  }%
}
\ExplSyntaxOff
```
(It checks whether the arguments is in the list and gives an error otherwise)

Reply:
I like that this new macro will refuse to make the url if the cip number is not recognized.  Nice touch @carlostome!

Strangely, if I define the macro without that test, it works fine, but when I add the `\clist` test `\href` insists on adding ".pdf" to the end of the URL! 🫤

Fortunately, I found a fix: `\detokenize`